src/renderers/dom/client/wrappers/ReactDOMSelect.js

```diff
@@ -236,7 +236,9 @@ function _handleChange(event) {
   var props = this._currentElement.props;
   var returnValue = LinkedValueUtils.executeOnChange(props, event);
 
-  this._wrapperState.pendingUpdate = true;
+  if (this._rootNodeID) {
```

Comment:
@jimfb Removed check for pendingUpdate. Let me know if there is anything else, or I misunderstood. Thanks!